src/system/data-transfer-tax-rates.md

```diff
@@ -6,74 +6,82 @@ If you conduct business in several states and ship a large quantity of product,
 
 ## Step 1: Export the Magento Tax Rate Data
 
-1.  On the _Admin_ sidebar, tap **System**.
+1. On the _Admin_ sidebar, go to **System** > _Data Transfer_ > **Import/Export Tax Rates**.
 
-1.  Under _Data Transfer_, choose **Import/Export Tax Rates**.
+1. Click **Export Tax Rates**.
 
-1.  Click **Export Tax Rates**. Look for the download file in the lower-left corner of the browser window.
+1. Look for the file in the download location for your web browser.
 
-1.  Save the file, and then open it in a spreadsheet. (In this example, we use OpenOffice Calc.)
+1. Save the file, and then open it in a spreadsheet.
+
+  This example uses OpenOffice Calc.
 
```

Comment:
this entire list is displaying as code, along with the image link, no image.
https://docs.magedevteam.com/294/m2/ee/user_guide/system/data-transfer-tax-rates.html

Reply:
fixed